src/javascripts/components/hub/hub.js

```diff
@@ -59,7 +58,7 @@ const editAirportEvent = (e) => {
     numShuttles: $('#edit-shuttles').val(),
     numHotels: $('#edit-hotels').val(),
     isInternational: radio,
-    uid: userId,
+    uid: firebase.auth().currentUser,
```

Comment:
shouldn't this have a uid at the end?

Reply:
I changed that in my last PR--- It is pull the user info into that variable 
will change now

Reply:
It should have been in there??

Reply:
done